platformio.ini

```diff
@@ -1228,6 +1228,28 @@ build_flags       = ${common_stm32.build_flags}
 extra_scripts     = ${common.extra_scripts}
   pre:buildroot/share/PlatformIO/scripts/generic_create_variant.py
 
+#
+# Anet ET4-MB_V1.x/ET4P-MB_V1.x (STM32F407VGT6 ARM Cortex-M4)
+# For use with with davidtgbe's OpenBLT bootloader https://github.com/davidtgbe/openblt/releases
+# Comment out board_build.offset = 0x10000 if you don't plan to use OpenBLT/flashing directly to 0x08000000.
+#
+[env:Anet_ET4_OpenBLT]
+#build_type = debug
+#debug_build_flags = -O0 -ggdb3 -g3
+platform           = ${common_stm32.platform}
+extends            = common_stm32
+board              = ANET_ET4
+board_build.offset = 0x10000
+build_flags        = ${common_stm32.build_flags}
+  -DDISABLE_GENERIC_SERIALUSB
+  -DSTM32F407VG -DARDUINO_ARCH_STM32 -DTARGET_STM32F4
+build_unflags      = ${common_stm32.build_unflags}
+  -DUSBCON -DUSBD_USE_CDC -DUSBD_VID=0x0483
+extra_scripts      = ${common.extra_scripts}
+  pre:buildroot/share/PlatformIO/scripts/copy_marlin_variant_to_framework.py
```

Comment:
maybe we should use our new generic script for variants `pre:buildroot/share/PlatformIO/scripts/generic_create_variant.py`.

Reply:
Let's! Per our discussion & testing via Discord, this seems to work fine. It'll be nice to clean up/dump all those extra files!